src/Backend/Core/Layout/Templates/FormLayout.html.twig

```diff
@@ -62,17 +62,27 @@
   {% endspaceless %}
 {%- endblock label_asterisk -%}
 
+{% block form_errors_list_content %}
+  <ul class="list-unstyled">
+    {%- for error in errors -%}
+      <li {% if form.parent %}class="formError"{% endif %}>{{ macro.icon('exclamation-triangle') }} {{ error.message }}</li>
+    {%- endfor -%}
+  </ul>
+{% endblock %}
+
 {% block form_errors -%}
   {% import 'Layout/Templates/macros.html.twig' as macro %}
+
   {% if errors|length > 0 -%}
-    {% if form.parent %}<small class="form-text text-muted">{% else %}<div class="alert alert-danger">{% endif %}
-    <ul class="list-unstyled">
-      {%- for error in errors -%}
-        <li {% if form.parent %}class="formError"{% endif %}>{{ macro.icon('exclamation-triangle') }} {{ error.message }}</li>
-      {%- endfor -%}
-    </ul>
-    {% if form.parent %}</small>{% else %}</div>{% endif %}
+    {% if form.parent %}
+      <small class="form-text text-muted">
+        {{ block('form_errors_list_content') }}
+      </small>
+    {% else %}
+      {{ macro.alert('danger', block('form_errors_list_content')) }}
+    {% endif %}
   {%- endif %}
+
```

Comment:
Definitely think it's a good idea, but the reason why i do not yet changed it is because we also still use spoon form for like half of our forms. I should wait to change it after the spoon forms are converted to symfony. Otherwise we will have 2 different types op checkboxes etc. And changing the spoon checkboxes layout at this point seems useless in my eyes. @sarahmey, @carakas what do you think?

Reply:
I don't really have an opinion on when the best time is to do this, but I would find it an improvement if it gets in fork at some point, so indeed, maybe ask carakas about it.

Reply:
I would already change it. We can always backport those changes to spoon if needed

Reply:
Ok I will change it for symfony forms